dependencies.yaml

```diff
@@ -142,6 +142,7 @@ dependencies:
           - cxx-compiler
           - fmt>=11.0.2,<12
           - libcumlprims==24.10.*,>=0.0.0a0
+          - libcuvs==24.10.*,>=0.0.0a0
```

Comment:
Do we still need `libraft` here or can we just rely on `libraft-headers`?

Reply:
I've removed libraft now, and we should only be using libraft-headers now